test/functional/apps/home/_home.js

```diff
@@ -26,7 +26,7 @@ export default function ({ getService, getPageObjects }) {
     });
 
     it('clicking on console on homepage should take you to console app', async () => {
-      await PageObjects.common.navigateToUrl('home');
+      await PageObjects.common.navigateToApp('home');
```

Comment:
:information_source: This would actually not be needed with further changes I made to the PR, since those URLs are now handled properly, but imho that looks anyway nicer, so I just left it in :)